Add mermaid diagram to show relations between the terms (#28)
asam-ev/qc-framework: doc/manual/architecture.md

    simulation. These are "driven" in dynamic tests.
  - OpenSCENARIO (XOSC): Scenario definition, includes route definition (one file per route with
    extension .xosc)
- **Rule:** A textual description/definition about the input file
  characteristics.

> - **Check/Checker:** A software module that addresses to check one or multiple
>   rules or creates statistical information from the input file model. This can
>   be a static or a dynamic test. For a list of possible tests, see parent page.

Reviewer: ```suggestion
- **Check/Checker:** A software module that tests rules compliance
  or creates statistical information about the input file. This can
  be a static or a dynamic test. For a list of possible tests, see parent page.
```
